macros/DekiScript-Page.ejs

```diff
@@ -1,8 +1,8 @@
 <% module.exports = buildAPI({
-    
+
```

Comment:
In the future, please split up the "normalize whitespace" and "make a change" commits. No need to redo this PR.